### DISCUSSION TITLE
Is it possible to make live transcriptions?

### DISCUSSION THREAD

**Discussion by ghost** [openai/whisper#636] [Q&A]
I have been trying to make a program that can transcribe and translate speech in real time. After many days of reserching, I have found nothing of use. Is this even posible?

**Comment by mahenrathod** [comment]
Whisper doesn't support live transcription. However, you can use Python websocket to get the stream of audio and transcribe in the chunk. You also need enough RAM to load the model and transcribe the chunk of audio in order to get the live transcriptions.

**Reply by ghost** [reply]
Thank you so much. You have no idea how helpful you have been.

**Reply by mahenrathod** [reply]
Thanks @Penguin3266 , That's what the community is all about.

**Comment by glangford** [comment]
> I have been trying to make a program that can transcribe and translate speech in real time. After many days of reserching, I have found nothing of use. Is this even posible?

See the ideas in https://github.com/openai/whisper/discussions/608 and https://github.com/openai/whisper/discussions/631 to build a layer around whisper to support live transcription.
